### PR TITLE
Remove Django Envelope

### PR DESCRIPTION
**Pull request by elmq0022**
Looking to remove django-envelope from the project due to lack of package support.  Pull request is mostly complete, and I just need to workout a few bugs and test.

### CODE REVIEW

**Comment by sophm256** [comment]
Although we're not storing the data in a database, perhaps putting a limit (max length) on how much you can enter into 'message' field for the Contact form would be good.

**Comment by sophm256** [comment]
Not sure if it's possible to put a limit on the captcha field, but putting a max length on 'sender', 'email', and 'subjects' (as well as 'message') would be probably be good defensive programming.

**Comment by sophm256** [comment]
'Captcha' field is not a CharField, just a checkbox, so putting on a limit on it doesn't apply here.

**Comment by elmq0022** [comment]
I think I got it.  You'll see an error message on the homepage after sending a message, but this will work fine in an actual deploy with an email back end setup.

**Comment by elmq0022** [comment]
It took me awhile digging through the code, but I finally found a way to make the recaptcha library play with the testing correctly.  Yay :smile:

File: chipy_org/apps/contact/forms.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            msg = EmailMessage(
+                subject=self.cleaned_data["subject"],
+                body=self.cleaned_data["message"],
+                from_email=self.cleaned_data["email"],
+                to=getattr(settings, "ENVELOPE_EMAIL_RECIPIENTS", []),
+                reply_to=[self.cleaned_data["email"]],
+            )
+            msg.send()
+            return True
+        except Exception as e:
```
Comment (raymondberg) [inline chipy_org/apps/contact/forms.py:33]: I don't love the bare except, but I'd MUCH rather have Envelope out.

Comment (elmq0022) [reply chipy_org/apps/contact/forms.py]: Took your advice and move the exception.  Let me know if you see anything else to change.

File: chipy_org/apps/contact/tests.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        "subject": "test subject",
+    }
+
+    form = ContactForm(form_data)
+    assert form.is_valid()
+    form.send_email()
+    assert len(mail.outbox) == 1
+
+
+@pytest.mark.django_db
```
Comment (sophm256) [inline chipy_org/apps/contact/tests.py:47]: Does this need to be here? No database data is being used.

**Comment by sophm256** [comment]
Some things to consider:

MAKE TEST RESULTS
I ran `make test` and 38 passed, 1 skipped . The skipped test was `chipy_org/apps/profiles/tests.py::SmokeTest::test__profile_edit_url__POST_auth SKIPPED [ 82%]` 

I'm guessing this does not have to do with your code.

.....................................................................................................................................

MAKE LINT RESULTS
```
Module chipy_org.apps.contact.tests
chipy_org/apps/contact/tests.py:17:23: W0621: Redefining name 'no_recaptcha' from outer scope (line 11) (redefined-outer-name)
chipy_org/apps/contact/tests.py:30:28: W0621: Redefining name 'no_recaptcha' from outer scope (line 11) (redefined-outer-name)
chipy_org/apps/contact/tests.py:49:36: W0621: Redefining name 'no_recaptcha' from outer scope (line 11) (redefined-outer-name)

-----------------------------------
Your code has been rated at 9.98/10
```
.............................................................................................................

'MAKE FORMAT-CHECK' RESULTS
```
docker-compose exec web black --diff .
--- chipy_org/settings_test.py	2020-07-11 00:30:13.509976 +0000
+++ chipy_org/settings_test.py	2020-07-11 00:50:05.839897 +0000
@@ -3,11 +3,13 @@
 
 DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": ":memory:", "TEST": {}}}
 DEBUG = True
 ADMINS = ["admin@chipy.org"]
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-ENVELOPE_EMAIL_RECIPIENTS = ["admin@example.com",]
+ENVELOPE_EMAIL_RECIPIENTS = [
+    "admin@example.com",
+]
 
 SECRET_KEY = "somesecretkeyfordjangogoeshere"
 SECURE_SSL_REDIRECT = False
 
 NORECAPTCHA_SITE_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
reformatted chipy_org/settings_test.py

1 file reformatted, 71 files left unchanged.
```
...................................................................................................................

**CONCLUSION**: Please fix linter error and also run `make format` on your code. Thanks!

File: chipy_org/apps/contact/tests.py (hunk 2 above)
Comment (elmq0022) [reply chipy_org/apps/contact/tests.py:47]: Actually yes.  The database is needed for this test.

**Comment by elmq0022** [comment]
I've corrected the lint errors and updated the black/isort formats.  Thanks! :cup_with_straw:

File: chipy_org/apps/contact/tests.py (hunk 2 above)
Comment (sophm256) [reply chipy_org/apps/contact/tests.py:47]: Just wondering, how or where is the database being used?

**Comment by sophm256** [comment]
Awesome!

**Pull request merged**